source/__tests__/new/source-profile.test.js

```diff
@@ -0,0 +1,18 @@
+describe('Sinopia Profile Source Field', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://localhost:8000/#/profile/create/')
+  })
+
+  it('finds in the profile', async () => {
+    const label = await page.$eval('#profile > div > table > tbody > tr:nth-child(4) > td:nth-child(3) > label', e => e.textContent)
+    await expect(label).toMatch(/Source/)
+    const field = await page.$eval('#profile > div > table > tbody > tr:nth-child(4) > td:nth-child(4) > input', e => e.getAttribute('name'))
+    await expect(field).toMatch(/source/)
+  })
+
+  it('mouse hover displays help', async () => {
+    const popover = await page.$eval('#profile > div > table > tbody > tr:nth-child(4) > td:nth-child(4) > input', e => e.getAttribute('popover'))
+    await expect(popover).toMatch(/Link to more information about profile/)
```

Comment:
similarly, this test could look for 

```
    const popover = await page.$eval('#profile  input[name="source"]', e => e.getAttribute('popover'))
```

and you could maybe even declare
```
let source_input_sel = '#profile input[name="source"]' 
```
within the describe block so you could use it for both tests